stub/src/main/java/io/grpc/stub/AbstractStub.java

```diff
@@ -121,6 +122,18 @@ public final S withDeadlineAfter(long duration, TimeUnit unit) {
   }
 
   /**
+   * Returns a new stub with the given executor that is to be used instead of the default one
+   * specified with {@link ManagedChannelBuilder#executor}. Note that setting this option may not
+   * take effect for blocking unary calls.
```

Comment:
Not just for unary calls, it's also the case for blocking server-streaming calls.
You may just say "for blocking calls", or "on blocking stubs".